standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -433,6 +433,7 @@ public void init() throws MetaException {
         addAdminUsers();
         currentUrl = MetaStoreInit.getConnectionURL(conf);
       }
+      TxnUtils.initializeTxnStore(conf); // Must be before Metric services start
```

Comment:
hmm will each `HMSHandler` be initialized in its own thread? curious how is this different from the existing code.

Reply:
Yea so if you trace HiveMetastore.startMetaStore() => RetryingHMSHandler.getProxy => HMSHandler.init() (so this makes it eager initialization) 

I thought putting it here fixes the test, but actually tests are still running so maybe there are other test errors after.

Reply:
Oh I see. Makes sense.

Strange that I don't see the latest test results.